values.yaml

```diff
@@ -103,6 +103,9 @@ proxy:
   remoteurl: https://registry-1.docker.io
   username: ""
   password: ""
+  # the ref for a secret stored outside of this chart
+  # proxyUsername and proxyPassword are the keys to include in the said secret
```

Comment:
A very minor suggestion to reduce verbosity on this line:

```
# Keys: proxyUsername, proxyPassword
```

Reply:
Alright updated :)